src/Api/Endpoints/Login.php

```diff
@@ -22,11 +22,21 @@
  * @category API
  * @package  Loris
  * @author   Dave MacFarlane <dave.macfarlane@mcin.ca>
- * @license  Loris license
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://github.com/aces/Loris
  */
 class Login extends Endpoint
 {
+    /* @var int A supplied JWT key must have at least this length or it will
+     * be rejected for being too weak.
+     */
+    protected const MIN_JWT_KEY_LENGTH = 20;
```

Comment:
missing a newline to separate the comments for the header below from the property above it

Reply:
Is this addressed with the newest changes? Not sure if I'm understanding you correctly.

Reply:
No.

I meant it's written

```
/**
 * blah blah blah
 */
 protected const foo;
 /*
  * blah blah blah
  */
 protected const bar;
```

Instead of:
```
/**
 * blah blah blah
 */
 protected const foo;

 /*
  * blah blah blah
  */
 protected const bar;
```

The newline separating the two is missing.

Reply:
Ok, thanks for clarifying